spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/table/SparkBaseTable.java

```diff
@@ -47,7 +48,7 @@ public abstract class SparkBaseTable implements Table, SupportsRead, SupportsWri
   private TableCatalog sparkCatalog;
   private Table lazySparkTable;
   private PropertiesConverter propertiesConverter;
-  private SparkTransformConverter sparkTransformConverter;
+  @Getter private SparkTransformConverter sparkTransformConverter;
```

Comment:
please add @VisibleForTesting

Reply:
Same as [this](https://github.com/datastrato/gravitino/pull/2709#discussion_r1573134516) above.

Reply:
updated.